Sources/_MatchingEngine/Engine/Tracing.swift

```diff
@@ -25,8 +36,16 @@ extension Operand: CustomStringConvertible {
   }
 }
 
-extension Processor.SavePoint: CustomStringConvertible {
-  var description: String {
-    String(describing: self.destructure)
+extension Processor.SavePoint {
+  func describe(in input: Input) -> String {
```

Comment:
```suggestion
  func description(in input: Input) -> String {
```
Nit: The method doesn't have side effects. Therefore suggesting a noun phrase instead of an imperative.